index.html

```diff
@@ -24,15 +24,45 @@
   <body>
     <header class="header">
       <nav id="navbar" class="navbar">
-        <a href="">Kennedy Owusu</a>
-        <a id="menu-icon" href=""><img src="./images/nav-icons/nav-icon.png"></a>
+        <a class="navbar-title" href="">Kennedy Owusu</a>
         <ul>
           <li><a class="header-nav" href="">Portfolio</a></li>
           <li><a class="header-nav" href="">About</a></li>
           <li><a class="header-nav" href="">Contact</a></li>
           <li><a class="header-nav" href=""><img src="./images/nav-icons/mail-icon.png"></a></li>
         </ul>
```

Comment:
It's highly recommended to hide this desktop menu on the mobile version, it'll help your design look more professional. So kindly make sure to fix it.

Reply:
This is the code for the desktop menu in the HTML (This code can be found on lines 26 - 35 in the index.html file)

`<nav id="navbar" class="navbar">

        <a class="navbar-title" href="">Kennedy Owusu</a>
        <ul>
          <li><a class="header-nav" href="">Portfolio</a></li>
          <li><a class="header-nav" href="">About</a></li>
          <li><a class="header-nav" href="">Contact</a></li>
          <li><a class="header-nav" href=""><img src="./images/nav-icons/mail-icon.png"></a></li>
        </ul>

      </nav>`

And this is me hiding the desktop menu in the CSS for the mobile version (this code can be found on lines 670 - 672 in the style.css file)

`#navbar {
    display: none;
  }`


I have tested the mobile version on both Chrome and Firefox and I don't see the desktop menu you are making reference to

These are mobile images of the project on different browsers.


This is for Chrome
![mobile-version-chrome](https://user-images.githubusercontent.com/32777408/202016065-c9ba815d-67d6-46cf-a95e-4e0bb493ea0a.png)

This is for Firefox
![mobile-version-firefox](https://user-images.githubusercontent.com/32777408/202016253-4eeac2b5-387d-4556-b67a-68195dd81745.png)

This is for Chrome with a max-width of 540px
![max-540](https://user-images.githubusercontent.com/32777408/202016316-214c812c-6c63-4fd8-a285-624166c5a816.png)

This is for Firefox with a max-width of 540px

![540-on-firefox](https://user-images.githubusercontent.com/32777408/202016777-1250bc43-4414-4bca-996d-fa32b9c7276d.png)